cmd/datamon/cmd/cli_test.go

```diff
@@ -307,6 +307,104 @@ func TestUploadBundle_filePath(t *testing.T) {
 	}, "observe error on bundle upload path as file rather than directory", true)
 }
 
+func getTempFile(t *testing.T, filename string) (*os.File, func()) {
```

Comment:
nitpick/information sharing: on internal test functions, we may use `t testing.TB` interface instead. This allows test utility funcs to be reused by benchmarks.

Reply:
noted